tetrateci/gen_release_manifest.py

```diff
@@ -27,8 +27,11 @@
     manifest["version"] = tag
     manifest["dependencies"]["istio"] = {"localpath" : "./istio"}
     manifest["dependencies"]["client-go"]["branch"] = branch
-    manifest["dependencies"]["gogo-genproto"]["branch"] = branch
     manifest["dependencies"]["tools"]["branch"] = branch
+    # genproto has been removed from 1.14
+    # added check for "gogo-genproto" dependenciy if it present then assign branch
+    if "gogo-genproto" in manifest["dependencies"]:
+        manifest["dependencies"]["gogo-genproto"]["branch"] = branch
```

Comment:
@psbrar99 added compatibility check for genproto lib. It would work with <1.14 release.